api/server/services/Artifacts/update.spec.js

```
@@ -0,0 +1,320 @@
+const {
+  ARTIFACT_START,
+  ARTIFACT_END,
+  findAllArtifacts,
+  replaceArtifactContent,
+} = require('./update');
+
+const createArtifactText = (options = {}) => {
+  const { content = '', wrapCode = true, isClosed = true, prefix = '', suffix = '' } = options;
+
+  const codeBlock = wrapCode ? '```\n' + content + '\n```' : content;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)